test/Npgsql.Tests/ConnectionStringBuilderTests.cs

```diff
@@ -127,4 +128,46 @@ public void Invalid_connection_string_throws()
         Assert.That(() => builder.ConnectionString = "Server=127.0.0.1;User Id=npgsql_tests;Pooling:false",
             Throws.Exception.TypeOf<ArgumentException>());
     }
+
+    [Test]
```

Comment:
Don't forget to add `NonParallelizable` here since you change environment variables.

Reply:
Done.
I have aligned the test on existing tests using also environment variables.